target/linux/mvebu/image/cortexa9.mk

```diff
@@ -32,8 +32,8 @@ define Device/cznic_turris-omnia
   DEVICE_VENDOR := CZ.NIC
   DEVICE_MODEL := Turris Omnia
   KERNEL_INSTALL := 1
-  KERNEL := kernel-bin
-  KERNEL_INITRAMFS := kernel-bin
+  KERNEL := kernel-bin | append-dtb
+  DEVICE_DTS := armada-385-turris-omnia
```

Comment:
DEVICE_DTS is still here and I don't see why it would need to be set explicitly ...

Reply:
Ouh god that is second time today. It wasn't pushed as it seems. (I am beginning to suspect something shifty about my fork).

Anyway it seems to be pushed now.